assets/style.css

```diff
@@ -0,0 +1,781 @@
+  }
+
+  /* SHOWCASE */
+  .showcase-section {
+    grid-template-columns: 70% 30%;
+    gap: 1em;
+    margin: 0 auto;
+    width: 99.9%;
+  }
+
+  .main-showcase {
+    height: 500px;
+    width: 100%;
+  }
+
+  .aside-showcase {
+    height: 243px;
+  }
+
+  .main-showcase > div h1 {
+    font-size: 40px;
+  }
+
+  .aside-showcase > div h1 {
+    font-size: 20px;
+  }
+
+  .aside-showcase > div :nth-last-child(1) {
+    display: none;
+  }
+
+  /* LATEST NEWS */
+  .latest-news-div {
+    flex-direction: column;
+  }
+
+  .latest-news-img {
+    width: 100%;
+  }
+
+  /* LATEST FUNDING-ROUNDS */
+  .latest-news-grid,
+  .latest-funding-contents {
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1em;
+  }
+
+  .width-768,
+  .width-320 {
+    display: none;
+  }
+
+  .funding-links-right {
+    flex-direction: column;
+    padding-left: 0;
+  }
+
+  /* MOST POPULAR */
+
+  .most-popular-section {
+    grid-template-columns: repeat(2, 1fr);
+    gap: 1em;
+  }
+
+  .hardfork-img {
+    height: 300px;
+    width: 100%;
+  }
+
+  /* FOOTER */
+  .footer {
+    width: 100%;
+    height: 250px;
+    text-align: center;
+  }
+
+  .footer-links {
+    background-color: #131313;
+    padding: 2.5em 0;
+  }
+
+  .wrapper {
+    margin: 0 auto;
+    max-width: 100%;
+    padding: 0 20px;
+    height: 100%;
+  }
+
+  .footer-links-social,
+  .footer-links-social li a,
+  .footer-links-menu {
+    justify-content: center;
+  }
+
+  .footer-links-social {
+    font-size: 100%;
+    margin-bottom: 15px;
+    vertical-align: baseline;
+  }
+
+  .footer-links-social li a {
+    box-sizing: border-box;
+    background-color: #333;
+    margin: auto 15px;
+    border-radius: 20px;
+    padding: 10px;
+  }
+
+  .footer-links-menu {
+    margin-top: 50px;
+    font-size: 100%;
+    box-sizing: border-box;
+    align-items: center;
+  }
+
+  .footer-links-menu li a {
+    margin: 5px;
+    border: 0;
+    vertical-align: baseline;
+    box-sizing: border-box;
+    align-content: center;
+    flex-wrap: nowrap;
+  }
+
+  .footer-legal {
+    width: 100%;
+    height: 100px;
+    padding: 50px 0;
+    background-color: #000;
+    display: block;
+  }
+
+  span img {
+    width: 37px;
+    height: 9px;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  /* ====== NAVIGATION ====== */
+  .logo-div,
+  .fa-bars {
+    display: none;
+  }
+
+  .width-1024 {
+    display: block;
+  }
+
+  .width-1023 {
+    display: block;
+  }
+
+  .width-320 {
+    display: none;
+  }
+
+  .first-navbar {
+    background-color: #131313;
+  }
+
+  .split-links,
+  .second-nav-links,
+  .latest-funding-title-links,
+  .footer-links {
+    justify-content: space-between;
+  }
+
+  .nav-flex {
+    width: 100%;
+  }
+
+  .funding-links-right li {
+    padding: 0 5px;
+    border-right: 1px solid #fe4736;
+  }
+
+  .first-navbar ul li {
+    margin-right: 10px;
+  }
+
+  .first-navbar li a {
+    color: #555;
+  }
+
+  .news {
+    color: #fff !important;
+  }
```

Comment:
## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

**Note:** Check all this entire page and fix them

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_